tests/integration/docusaurus/expectations/advanced/data_assistant_cross_table_comparison.py

```diff
@@ -64,7 +64,17 @@
 # </snippet>
 # <snippet name="tests/integration/docusaurus/expectations/advanced/data_assistant_cross_table_comparison.py run_assistant">
 data_assistant_result = context.assistants.onboarding.run(
-    batch_request=pg_batch_request, exclude_column_names=["VendorID"]
+    batch_request=mysql_batch_request,
+    exclude_column_names=[
+        "VendorID",
+        "improvement_surcharge",
```

Comment:
Is there a reason we are changing the batch request from `pg` to `mysql`? Also, what was the reason for changing the `exclude_column_names`?

Reply:
@Shinnnyshinshin Yes, the `pg` as a typo, which had to be corrected.  The `exclude_column_names` is needed because the numerical calculations between the two database backends are not the same.  Thank you.

Reply:
We're dealing with 2 different batch requests, and one of them was used in the wrong place for the flow of the script.

Additionally, we're excluding some columns to avoid failing validations that were causing the script to fail.